### PR TITLE
Add meetings menu item and description

### PR DESCRIPTION
**Pull request by graeme-a-stewart**
We now run a few series of meetings that are neither
working groups per-se or activities. It seemed then best
to add these are a separate set of topics, using the same
Jekyll engine way to process them automatically.

A new dropdown menu link is then added to the navbar
for these items.

### CODE REVIEW

File: _meetings/roundtable.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+title: "Software and Computing Roundtable"
+author: "Graeme Stewart"
+layout: plain
+---
+
+<p align=center>
+<img src="/images/Roundtable-banner.jpg"
+  alt="SIDIS-logo" style="display: inline-block; width:80%; margin:3%">
```
Comment (eduardo-rodrigues) [inline _meetings/roundtable.md:9]:
```suggestion
  alt="Roundtable-logo" style="display: inline-block; width:80%; margin:3%">
```

File: _meetings/roundtable.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+</p>
+
+The *Software and Computing Roundtable* is a general series of meetings on topical
+matters for the high-energy and nuclear physics community, which is organised
```
Comment (eduardo-rodrigues) [inline _meetings/roundtable.md:13]:
```suggestion
matters for the nuclear and high-energy physics community, which is organised
```

Comment (eduardo-rodrigues) [reply _meetings/roundtable.md:13]: Am suggesting this swap since the roundtable comes from the NP community.

File: _meetings/roundtable.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+The Roundtable has historically had particularly strong links to the nuclear
+physics community, but is now expanding to be the perfect meeting point
+with high-energy physics.
+
+* [Roundtable Home](https://www.jlab.org/software-and-computing-round-table)
+* [Roundtable Meetings in 2021](https://indico.jlab.org/event/420/)
+* [Roundtable Meetings in 2020](https://indico.jlab.org/event/356/)
+
+If you would like to suggest a topic for the Roundtable, please contact [HSF
```
Comment (eduardo-rodrigues) [inline _meetings/roundtable.md:25]:
```suggestion
If you would like to suggest a topic for the Roundtable, please contact the [HSF
```

File: _meetings/roundtable.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+Coordination](mailto:hsf-coordination@googlegroups.com). The current HSF
+representative is [Graeme Stewart](mailto:graeme.andrew.stewart@cern.ch).
+
+<p align=center>
+<img src="/images/JLab_logo.png"
+  alt="SIDIS-logo" style="display: inline-block; width:25%; margin:3%">
```
Comment (eduardo-rodrigues) [inline _meetings/roundtable.md:31]:
```suggestion
  alt="JLab-logo" style="display: inline-block; width:25%; margin:3%">
```

File: _meetings/roundtable.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+<img src="/images/hsf_logo_angled.png"
+  alt="HSF-logo" style="display: inline-block; width:20%; margin:3%">
+<img src="/images/BNL-logo.png"
+  alt="openlab-logo" style="display: inline-block; width:25%; margin:3%">
```
Comment (eduardo-rodrigues) [inline _meetings/roundtable.md:35]:
```suggestion
  alt="BNL-logo" style="display: inline-block; width:25%; margin:3%">
```

File: _meetings/roundtable.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+If you would like to suggest a topic for the Roundtable, please contact [HSF
+Coordination](mailto:hsf-coordination@googlegroups.com). The current HSF
+representative is [Graeme Stewart](mailto:graeme.andrew.stewart@cern.ch).
+
+<p align=center>
+<img src="/images/JLab_logo.png"
+  alt="SIDIS-logo" style="display: inline-block; width:25%; margin:3%">
+<img src="/images/hsf_logo_angled.png"
+  alt="HSF-logo" style="display: inline-block; width:20%; margin:3%">
+<img src="/images/BNL-logo.png"
```
Comment (eduardo-rodrigues) [inline _meetings/roundtable.md:34]: Do you have the BNL logo file somewhere already, or it is missing in this PR?

**Review by eduardo-rodrigues — CHANGES_REQUESTED** [review]
Some quick fixes to implement :-).

File: _meetings/roundtable.md (hunk 6 above)
Comment (graeme-a-stewart) [reply _meetings/roundtable.md:34]: It's already there as they have had GSoC projects in the past

**Comment by graeme-a-stewart** [comment]
Thanks for spotting all my cut'n'paste errors @eduardo-rodrigues (and apologies for having them there).

**Review by eduardo-rodrigues — APPROVED** [review]

**Pull request merged**